.gitignore

```diff
@@ -1,6 +1,54 @@
-target/
+/target/
+.gradle
+build/
+!**/src/main/**/target/
+!**/src/test/**/target/
+!**/src/main/**/build/
+!**/src/test/**/build/
 
-# The following files are generated/updated by Hilla
+.DS_Store
+
+# The following files are automatically generated/updated
 node_modules/
 frontend/generated/
+pnpmfile.js
+.npmrc
+webpack.generated.js
 vite.generated.ts
+
+# Eclipse and STS
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+# IntelliJ IDEA
+.idea
+*.iws
+*.iml
+*.ipr
+out/
+!**/src/main/**/out/
+!**/src/test/**/out/
+
+# NetBeans
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+
+# VS Code
+.vscode/
+
+# Maven wrapper
+.mvn/
```

Comment:
It's also ignored by projects from https://start.spring.io/.

Reply:
The whole point of the Maven wrapper is to have it in the repository so people without Maven can start the project. If it should not be in the repo, let's remove it rather than ignore it?

Reply:
I disagree, but it's not worth debating. Removed.